src/main/frontend/components/page.css

```diff
@@ -326,6 +337,14 @@ html.is-native-ios {
       }
     }
   }
+
+  .block-content-wrapper {
```

Comment:
Why do we need to specify the `width` here?

Reply:
If not, the mirror code block exceeds the screen edge on iPhone.

Reply:
How about limiting the scope to mobile-only? It'll be less dangerous if we change the dom structure in the future.

Reply:
It's already mobile-only. codes (L320-L321) does this.